### ISSUE TITLE
WebApp not working on ARM / M1 pro CPU

### ISSUE THREAD

**Issue by mve** [EdwinVW/pitstop#119]
I tried to run all services in a Kubernetes cluster on my Macbook with an m1 pro chip. I built all images locally, all services start up except the webapp service.
The pod is showing the following error:

```
Unhandled exception. System.IO.FileNotFoundException: Could not load file or assembly '/Pitstop.WebApp.dll'. The system cannot find the file specified.
File name: '/Pitstop.WebApp.dll'
```

There is a file with that exact name in the root directory of the container.
Running that file with dotnet after i exec into the container throws the same error.

The strange thing is that if i run ``` dotnet publish -c Release -o out ``` on my system (Not in a container) and then run the dll on my system the webapp will work as expected. So it's specifically not working in Docker it seems.

**Comment by mve** [comment]
I have created a PR that seems to fix the issues I was having.
https://github.com/EdwinVW/pitstop/pull/120

**Comment by EdwinVW** [comment]
Fixed by PR #120. 
Thanks @mve!
